docs/en/setup/service-agent/java-agent/Plugin-list.md

```diff
@@ -136,3 +136,4 @@
 - undertow-worker-thread-pool
 - tomcat-thread-pool
 - guava-eventbus
+- shenyu-2.4.x （Please must active optional spring-webflux-5.x-plugin plugin when you install agent in shenyu）
```

Comment:
This file is used for CI check, please don't add comments, it must match the key in plugin.def file.

Reply:
Thank you. It has been fixed.